backend/handler_funcs.py

```diff
@@ -1,29 +1,11 @@
 import json
 
```

Comment:
this whole file is _pretty_ much the reason I added flask and can probably be wholesale delete